remote-sr/src/main/resources/chrome_profile/Default/Preferences

```diff
@@ -0,0 +1,68 @@
+{
+  "profile": {
+    "avatar_bubble_tutorial_shown": 2,
+    "avatar_index": 26,
+    "blacklisted_credentials_stripped": true,
+    "content_settings": {
+      "exceptions": {
+        "media_engagement": {
+          "http://localhost:8080,*": {
```

Comment:
this only works in dev environment

Reply:
Yes. We need to get the port from the http-server somehow.